recipes/install-ua-harvester.rb

```diff
@@ -0,0 +1,77 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-install-ua-harvester
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+engage_node = search(:node, 'role:engage').first
+
+elk_info = get_elk_info
+
+harvester_release = elk_info[:harvester_release]
+rest_auth_info = get_rest_auth_info
+stack_name = stack_shortname
+sqs_queue_name = "#{stack_name}-user-actions"
+region = node["opsworks"]["instance"]["region"]
+loggly_info = node.fetch(:loggly, { token: '', url: '' })
```

Comment:
you're talking about the just the loggly_info (also used in the rsyslog setup), right?